pkg/process/v5/transformers/os/transform.go

```diff
@@ -215,16 +215,19 @@ func deriveConstraintFromFix(fixVersion, vulnerabilityID string) string {
 	return constraint
 }
 
-func enforceConstraint(constraint, format, vulnerabilityID string) string {
-	constraint = common.CleanConstraint(constraint)
-	if len(constraint) == 0 {
+func enforceConstraint(fixedVersion, vulnerableRange, format, vulnerabilityID string) string {
+	if len(vulnerableRange) > 0 && !strings.HasSuffix(vulnerabilityID, "ALASKERNEL") {
```

Comment:
it looks like there is new special casing for `ALASKERNEL`, but I don't see how that relates to this (maybe I'm missing something)?

Reply:
ALASKERNEL generates it's own version range from a hard-coded lower bound for a given kernel:

https://github.com/anchore/grype-db/blob/dcc9d3f0136cd79a3eff3e4c123c98bc5fbed7a1/pkg/process/v5/transformers/os/transform.go#L200

this special casing is to avoid skipping that if vunnel ends up emitting a version range.

Reply:
Is there an existing test that covers this? if not, can you add a new test for this?